org.webonnix.beamprivace.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: org.beamprivacy.desktop
+runtime: org.kde.Platform
+runtime-version: '3.1.5765'
+sdk: org.kde.Sdk
+command: beamprivacy 
+rename-icon: beamprivacy 
+rename-desktop-file: beamprivacy.desktop
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host # write output files
+  - --share=network
+
+modules:
+- name: beamprivacy  
+  buildsystem: qmake
+  sources:
+    - type: git
+      url: https://github.com/BeamMW/beam.git
+      branch: mainnet
```

Comment:
As we want builds to be reproducible, you should use specific commit or tag here.